tests/generators/OperationGenerator.ts

```diff
@@ -323,7 +317,7 @@ export default class OperationGenerator {
    */
   public static async createUpdateOperationRequest (
     didUniqueSuffix: string,
-    updateRevealValue: string,
+    updateKey: JwkEs256k,
     nextUpdateCommitmentHash: string,
     patches: any,
     signingKeyId: string,
```

Comment:
Sanity: I thought you mentioned you were going to remove the key ID? Which I think is the right thing.